CONTRIBUTING.md

```diff
@@ -1,53 +1,121 @@
-## How to Contribute
+## How to Contribute  
 
-1. Fork this repository to your github account.
+1. Fork this repository to your github account.  
 
 2. Clone the forked repository.  
-    ```
-    git clone https://github.com/<github-username>/STAC-IITMandi.github.io.git
-    ```
-3. Database Setup
+   `git clone https://github.com/<github-username>/STAC-IITMandi.github.io.git`  
 
-    1. You should have postgresql installed on your system. If not, run  
-        ```
-        sudo apt update
-        sudo apt install postgresql postgresql-contrib
+3. Database Setup  
+   1. You should have postgresql installed on your system. If not, run    
+    
+      **On Linux** -    
+        `sudo apt update`  
+
+        `sudo apt install postgresql postgresql-contrib`  
+
+      **On Windows** -     
+            Install -     
+                  1. PostgreSQL - `https://www.postgresql.org/download/windows/`. 
+   
+                        **( Note :- Remeber Password entered here.)**.  
+                                 
+                  2. pgAdmin - `https://www.pgadmin.org/download/pgadmin-4-windows/`  
```

Comment:
The formatting could be improved, this is due to the newlines and backticks. Try changing them and backticks could be removed from the URLs.

Reply:
improved.